empress/recon_vis/recon_viewer.py

```diff
@@ -150,7 +151,7 @@ def _render_host_helper(fig: plot_tools.FigureWrapper, node: tree.Node, show_int
         _render_host_helper(fig, node.right_node, show_internal_labels, font_size, host_tree)
 
 
-def _render_parasite(fig: plot_tools.FigureWrapper, parasite_tree: tree.Tree, recon_obj: recon.Reconciliation, host_lookup: dict, parasite_lookup: dict, show_internal_labels: bool, show_freq: bool, font_size: float):
+def _render_parasite(fig: plot_tools.FigureWrapper, parasite_tree: tree.Tree, recon_obj: recon.Reconciliation, host_lookup: dict, parasite_lookup: dict, show_internal_labels: bool, show_freq: bool, font_size: float, longest_host_name: int):
```

Comment:
I'm not sure why this function exists. Couldn't we just call the helper directly using parasite_tree.root?